src/Features/Core/Portable/Completion/CommonCompletionUtilities.cs

```diff
@@ -121,9 +120,8 @@ public static bool IsStartingNewWord(SourceText text, int characterPosition, Fun
             {
                 case SymbolKind.Method:
                 case SymbolKind.NamedType:
-                    if (symbols.Count > 1)
+                    if (overloadCount > 0)
```

Comment:
why did this change?

Reply:
I created a shared helper out of this method which take the count instead of symbols